reporting/plotting.jl

```diff
@@ -0,0 +1,85 @@
+using SystemBenchmark
+using CSV
+using DataFrames
+using Plots
+using Statistics
+gr()
+
+function plotreport(df,scale=2.0)
+    
+    platforms = ["Windows", "macOS", "Linux (x86", "Linux (aarch"]
```

Comment:
Any reason why they are called `Linux (x86` and `Linux (aarch`?

Reply:
It's me being lazy catching the difference in Linux strings in the OS field. Maybe the quick fix would be to remove the '(' chars

Reply:
Fixed